src/REST/Section/Project.php

```diff
@@ -31,6 +31,21 @@ public function get($project, array $expand = [])
         return $this->Jira->get("project/{$project}", $parameters);
     }
 
+    /**
+     * Returns all projects which are visible for the currently logged in user.
+     * If no user is logged in, it returns the list of projects that are visible when using anonymous access.
+     *
+     * @see https://docs.atlassian.com/software/jira/docs/api/REST/7.6.1/#api/2/project
+     *
+     * @return array
+     *
+     * @throws \Badoo\Jira\REST\Exception
+     */
+    public function getAll() : array
```

Comment:
I think `list()` name for this method would be more appropriate because of other sections. See `REST\Section\Field` and `REST\Section\Comment`. I tried to keep the same naming style across all classes in this code, because I believe this makes library usage easier. You just know what to expect from the new thing you had not tried yet if you already had an experience with other parts of the library.